tutorials/eve/pythia_display.C

```diff
@@ -228,7 +233,7 @@ void pythia_next_event()
    gTrackList->MakeTracks();
 
 
-   TEveElement* top = gEve->GetCurrentEvent();
+   TEveElement* top = (TEveElement *)gEve->GetCurrentEvent();
```

Comment:
`static_cast` is safer than a `reinterpret_cast`, if possible:

```suggestion
   TEveElement* top = static_cast<TEveElement *>(gEve->GetCurrentEvent());
```